src/firewheel/cli/configure_firewheel.py

```diff
@@ -178,6 +210,29 @@ def define_set_parser(self) -> argparse.ArgumentParser:
         )
         return parser
 
+    def _update_nested_dict(self, original: dict, updates: dict) -> dict:
+        """
+        This function recursively updates the original dictionary with values
+        from the updates dictionary. If a key in the updates dictionary is a
```

Comment:
Maybe we use code syntax since `updates` is a specific keyword argument?

```suggestion
        from the updates dictionary. If a key in the ``updates`` dictionary is a
```